lib/oppen.rb

```diff
@@ -94,72 +102,74 @@ def trim_trailing_whitespaces? = @trim_trailing_whitespaces
 
     def upsize_stack? = @upsize_stack
 
-    # Default config for Oppen usage
+    # Default config for Oppen usage.
+    #
     # @return [Config]
     def self.oppen
       new
     end
 
-    # Default config for Wadler usage
+    # Default config for Wadler usage.
+    #
     # @return [Config]
     def self.wadler(eager_print: true, trim_trailing_whitespaces: true, upsize_stack: true)
-      new(indent_anchor: IndentAnchor::ON_BEGIN, eager_print:, trim_trailing_whitespaces:, upsize_stack:)
+      new(eager_print:, indent_anchor: IndentAnchor::ON_BEGIN, trim_trailing_whitespaces:, upsize_stack:)
     end
   end
 
   # @param value [String]
-  # @param width [Integer] token width that defaults to value.length
+  # @param width [Integer] token width that defaults to value.length.
   #
-  # @return [Oppen::Token::String] a new String token
+  # @return [Token::String] a new String token.
   def self.string(value, width: value.length)
     Token::String.new(value, width:)
   end
 
   # @see Token::Whitespace
```

Comment:
Move the `@see` to the end of the documentation.

We still need a word on what this call. Do not copy what's in the class def.

Reply:
And the comment on `@see` applies everyewhere.